Makefile

```diff
@@ -143,8 +142,8 @@ deps: $(BUF) $(PROTOC) $(PROTOC_GEN_GO) $(PROTOC_GEN_GRPC_GATEWAY)
 
 .PHONY: local
 local: $(BUF)
-	buf check lint
-	buf check breaking --against-input '.git#branch=master'
+	buf lint
+#   buf breaking --against '.git#branch=master'
```

Comment:
Temporary disabled, since `master` doesn't include a valid `buf.yaml` file and otherwise this check fails